main/python-libseccomp/template.py

```diff
@@ -1,7 +1,7 @@
 # update libseccomp alongside this
 pkgname = "python-libseccomp"
 pkgver = "2.5.5"
-pkgrel = 0
+pkgrel = 1
```

Comment:
this doesn't need bumping

Reply:
Well, it links again the lib rebuilt in `libseccomp` and the comment reminds to keep the two versions in sync so I guess it is actually better to bump `pkgrel` in sync as well.

Reply:
it's not

Reply:
the version didn't change so there is nothing to keep in sync

Reply:
ok, it's functionally a noop anyway so I won't be arguing over it